bench/wtperf/wtperf.c

```diff
@@ -685,6 +698,33 @@ worker(void *arg)
             goto err; /* can't happen */
         }
 
+        /* Update the index-like table. */
+        if (opts->index_like_table && (*op != WORKER_READ && *op != WORKER_TRUNCATE)) {
```

Comment:
Technically speaking, a truncate operation would imply lots of deletions from the index.  I understand why we wouldn't choose to do this, but it warrants a comment.

Reply:
Added.